src/commands/ptal.ts

```diff
@@ -80,22 +102,68 @@ const generateReplyFromInteraction = async (description: string, github: string,
 				.setURL(githubURL.href);
 
 			components.push(githubLink);
+			
+			const [owner, repo, id] = pathSections;
```

Comment:
There us tge string "pull" in between the repo and the pr number. ID would always be "pull" here